src/stores/reservation.js

```diff
@@ -3,13 +3,18 @@ import { retrieveSeance } from "@/api/useSeancesApi.js";
 import { retrieveHall } from "@/api/useHallsApi.js";
 import { retrieveMovie } from "@/api/useMoviesApi.js";
 import { formatMovieLength } from "@helpers/useFormatMovieLength";
+import { saveReservation } from "@/api/useReservationApi";
+import $router from "@/router";
 
 export const useReservationStore = defineStore({
   id: "reservation",
   state: () => ({
     seance: null,
     hall: null,
     movie: null,
+    selectedSeatsRaw: [],
```

Comment:
I think you are only saving a value here but I'm not sure if it's used anywhere? 🤔
Do you use this in components?

Reply:
It's used in HallPlan.vue to restore selectedSeats if the user goes from defining ticket types back to selecting seats.

Reply:
Okaaay got it, didn't see it used in the store and the named gave me the idea it miiiight be something unused 😄

Reply:
Yeah, it's not a perfect solution, hence the strange naming 😄
I could make a function which would extract seat numbers from reservationStore.selectedSeats (array of objects with ticket data as well) instead of creating this variable.

Reply:
Sounds like a getter potential 👀

Reply:
Exactly, already done!